spring-pulsar-boot-autoconfigure/src/test/java/org/springframework/pulsar/autoconfigure/PulsarAutoConfigurationTests.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.pulsar.autoconfigure;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Ok so this is WEIRD.... the checkstyle has the following:
```xml
<module name="ImportOrder">
  <property name="groups" value="java,/^javax?\./,org,org.springframework,*"/>
  <property name="ordered" value="true"/>
  <property name="separated" value="true"/>
  <property name="option" value="top"/>
  <property name="sortStaticImportsAlphabetically" value="true"/>
</module>
```

It says `sortStaticImportsAlphabetically` but that is not what it does... it requires them to be at the very top before any other imports. It took me a good 30mins fighting this. Funny enough, IDEA using Spring Boot build conventions attempts to group the statics at the bottom and that is where they seem to make the most sense to me. Anyways... if you run into this @sobychacko please remember this strangeness and save 30mins of your life **NOT** batting the `ImportOrder` checkstyle failure.